VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -30,5 +30,8 @@ public ActionResult Contact()
         public string Hello() {
         return "this is not on the coding rules.";
         }
+        
+        private static string ChangePassword = "PassWord917231";
```

Comment:
[CODE_SMELL] Remove this method and declare a constant for this value.

Reply:
/workitem

Reply:
WorkItem 773 Created see [workItem](https://dev.azure.com/csedevops/ea38b5b8-2b4f-4721-8514-fe765aa4a23a/_workitems/edit/773).